search/engine.go

```diff
@@ -477,6 +488,14 @@ func (e *ESEngine) DoSearch(ctx context.Context, query Query, sortBy string, fro
 		}
 	}
 
+	tweetsByLang := <-tweetsByLangChannel
+	for lang, tweets := range tweetsByLang {
+		if _, ok := resultsByLang[lang]; !ok {
+			resultsByLang[lang] = make([]*elastic.SearchResult, 0)
+		}
+		resultsByLang[lang] = append(resultsByLang[lang], tweets...)
```

Comment:
Not sure I follow.

All tweets should be as one result, why you append them as individual results?
What is the difference from searching this with all other result types?

Reply:
This is one result per language, right?

Then the map should be  (not a list just pointer to result) is that correct?
`tweetsByLangChannel := make(chan map[string]*elastic.SearchResult)`

Reply:
fixed